dense_mapping_node:
  logs_folder: "/home/andresaguiar/Documents/vineslam_logs/"

  multilayer_mapping:
    # Datum
    datum:
      latitude: 41.2055204
      longitude: -8.307482
      altitude: 255.632

    # Grid map definitions
    grid_map:
      origin:
        x: -110.0
        y: -110.0
        z: -18.0
      width: 220.0
      lenght: 220.0
      height: 25.0
      resolution: 0.25 # meters
      output_folder: "/home/andresaguiar/Desktop/"